code/modules/unit_tests/species_change_organs.dm

```diff
@@ -0,0 +1,46 @@
+/**
+ * Unit test to ensure that, when a mob changes species,
+ * certain aspects are carried over between their old and new set of organs
+ * (brain traumas, cybernetics, and organ damage)
+ */
+/datum/unit_test/species_change_organs
+
+/datum/unit_test/species_change_organs/Run()
+	var/mob/living/carbon/human/dummy = allocate(/mob/living/carbon/human)
+
+	// Give a trauma
+	dummy.gain_trauma(/datum/brain_trauma/severe/blindness)
+	// Give a cyber heart
+	var/obj/item/organ/internal/heart/cybernetic/cyber_heart = new()
```

Comment:
`allocate`?

Reply:
all organs are stored in nullspace.

Reply:
It should be allocated anyways since we shouldn't assume anything like this in a test where we're fundamentally testing behavior that could be bugged